helpers/middleman/partners.rb

```diff
@@ -38,6 +40,10 @@ def categories_collection(lang = default_locale_obj[:lang])
 
   # localize_value should always have a third argument: fallback_locale!
   def partners_categories(lang = default_locale_obj[:lang], withIndex: false)
+    @partners_categories ||= {}
+    @partners_categories[lang] ||= {}
+    return @partners_categories[lang][withIndex] if @partners_categories[lang][withIndex]
```

Comment:
This should be `@partners_categories[lang][!!withIndex]` actually, as for hashes `hash[nil]` is not the same as `hash[false]`.